### PR TITLE
Fix 'Auto-indentation bug'

### PR DESCRIPTION
**Pull request by idbartosz**
Fix 'Auto-indentation bug' https://github.com/atom/language-sass/issues/173

### CODE REVIEW

**Comment by 50Wliu** [comment]
Please update the comments as well.

**Comment by idbartosz** [comment]
Done.

File: settings/language-sass.cson [hunk 1]
```diff
@@ -2,13 +2,13 @@
   'editor':
     'autoIndentOnPaste': false
     'increaseIndentPattern': '''(?x)
-      ^\\s* # Start of line
+      ^(?!\\s*$)\\s* # Start of non empty line
       (
-        [^:]+ # No colons (for regular selectors like body)
+        [^:@,]+ # No colons, commas and at signs (for regular selectors like body)
```
Comment (50Wliu) [inline settings/language-sass.cson]: Why don't we indent after a `@`?  Is this to capture the at-rule later on?

Comment (idbartosz) [inline settings/language-sass.cson]: Yes, line 7 prevents at-rule to be qualified as regular selector (all regular selectors get indent). Line 11 detects proper at-rule and decides about its indentation.

**Comment by 50Wliu** [comment]
Thanks!

**Pull request merged**